cumulus_etl/deid/codebook.py

```diff
@@ -242,16 +242,16 @@ def save(self, codebook_dir: str) -> bool:
         """
         saved = False
 
-        if self.modified:
-            logging.info("Saving codebook to: %s", codebook_dir)
-
+        if self.settings_modified:
             codebook_path = os.path.join(codebook_dir, "codebook.json")
             common.write_json(codebook_path, self.settings)
+            self.settings_modified = False
+            saved = True
```

Comment:
Before this, any modification of the cached Patient/Encounter mappings would write out the codebook settings all over again, and these days the codebook settings never change.

(In BCH's legacy i2b2 environment, this meant writing out 1G every time we had a new mapping... :frowning: )